src/components/common/Error/index.js

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import { Container, ErrorMessageContainer } from './styledComponents';
+
+const Error = (props) => {
+  return (
+    <Container>
+      <ErrorMessageContainer>
+        <h1>404 Page Not Found</h1>
```

Comment:
We can customize this error code and based on that can show a message..
You can pass error code in props and can utilize errorcode fr config to show message